src/eepp/ui/doc/syntaxdefinitionmanager.cpp

```diff
@@ -1683,7 +1807,7 @@ SyntaxDefinitionManager::SyntaxDefinitionManager() {
 	if ( ms_singleton == nullptr )
 		ms_singleton = this;
 
-	mDefinitions.reserve( 84 );
+	mDefinitions.reserve( 85 );
```

Comment:
This will end up being 86 if we merge both langs, can you set this one to 86 and I'll merge this PR after the other one? Thanks